README.md

```diff
@@ -133,6 +132,17 @@ docker build -t rac .
 docker run -p 3000:3000 rac
 ```
 
+### Getting the analyst email yourself
+
+It is often useful to quickly see what the analyst email looks like. If the `LDAP_UID` environment variable is not set and the user provides their email address on the contact info screen then the server will send the analyst email unencrypted to the user. This is solely for development purposes. Note that the review apps are configured this way.
+
+To do this locally:
+
+- ensure that `LDAP_UID` is not set in your terminal environment nor in your `.env` file
+- run `npm run prod`
+- fill out the report and include your email address on http://localhost:3000/contactinfo
```

Comment:
Could we configure a local env variable to a specific email? so that we can test anonymous flows, and other flows without filling out the email field?

Reply:
See PR #1788